src/api/Http/Urls.php

```diff
@@ -46,6 +46,8 @@ public function __construct(array $patternsWithWildcards = [])
      */
     public static function sanitize($url, $dropUserInfo = false)
     {
+        $url = (string)$url;
```

Comment:
Is this just being cautious or was this necessary for something specific?

Reply:
preg_replace is not too happy on PHP 8.2 if an object with __toString() gets passed to it.